testsuite/features/support/twopence_init.rb

```diff
@@ -112,7 +112,10 @@ def twopence_init(target)
   $rhlike_minion = twopence_init("ssh:#{ENV['RHLIKE_MINION']}") if ENV['RHLIKE_MINION']
   $deblike_minion = twopence_init("ssh:#{ENV['DEBLIKE_MINION']}") if ENV['DEBLIKE_MINION']
   $build_host = twopence_init("ssh:#{ENV['BUILD_HOST']}") if ENV['BUILD_HOST']
-  $nodes += [$minion, $ssh_minion, $rhlike_minion, $deblike_minion, $build_host]
+  # TODO: fix the next 2 lines and use ENV[]
+  $sles_salt_migration = twopence_init('ssh:sles-salt-migration.mgr.prv.suse.net')
+  $leap_salt_migration = twopence_init('ssh:leap-salt-migration.mgr.prv.suse.net')
```

Comment:
Yeah, mandatory to fix this before merging.

Reply:
Fixed.